src/apps/vpn/ui/screens/getHelp/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -126,6 +126,19 @@ MZViewBase {
             }
         }
 
+        MZCheckBoxRow {
+            id: checkBoxRowVPNSessionPingTimeout
+
+            Layout.rightMargin: MZTheme.theme.windowMargin
+            labelText: "VPNSession ping timeout debug mode"
+            subLabelText: "Shortens the VPNSession ping timeout from 3 hours to 2 minutes"
+            isChecked: MZSettings.vpnSessionPingTimeoutDebug
+            showDivider: false
+            onClicked: {
+                MZSettings.vpnSessionPingTimeoutDebug = !MZSettings.vpnSessionPingTimeoutDebug
+            }
+        }
```

Comment:
Fancy, I like it.

Reply:
This requires a reset to be applied right?

Reply:
A reset of the VPN / controller, yes - not the whole app. Do you think it would be better to just make a note in the dev menu or make the adjustment while the controller is on?